src/lib.rs

```diff
@@ -15,6 +15,28 @@ use pgmq_crate::query::{
 };
 use thiserror::Error;
 
+extension_sql!(
+    "
+    CREATE TABLE IF NOT EXISTS public.pgmq_meta (
+      queue_name VARCHAR UNIQUE NOT NULL,
+      created_at TIMESTAMP WITH TIME ZONE DEFAULT (now() at time zone 'utc') NOT NULL
+    );
```

Comment:
I think we won't be able to use the `IF NOT EXISTS` part since pgmq does not own the existing tables. Might be tricky, I'll have to think how we could handle that.